internal/host/host.go

```diff
@@ -240,6 +241,20 @@ func (m *Manager) populateDisksEligibility(inventoryString string) (string, erro
 	return string(result), nil
 }
 
+func (m *Manager) HandlePrepareInstallationFailure(ctx context.Context, h *models.Host, reason string) error {
```

Comment:
isn't it the same as `HandleInstallationFailure`?

Reply:
It adds the reason string.